### PR TITLE
feat(cb2-8089): use V3 in VTM

### PR DESCRIPTION
**Pull request by owen-corrigan-bjss**
CB2-8089: sets reason for creation on the new record created by amend VRM for the audit trail

### CODE REVIEW

File: src/processors/processVrmRequest.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const processPatchVrmRequest = (currentRecord: TechRecordGet, userDetails
     const updatedNewRecord = setCreatedAuditDetails(newRecord, userDetails.username, userDetails.msOid, new Date().toISOString(), currentRecord.techRecord_statusCode as StatusCode);
     (updatedNewRecord as TechRecordHgv | TechRecordMotorcycle | TechRecordCar | TechRecordPsv).primaryVrm = newVrm.toUpperCase();
     const updatedRecordToArchive = setLastUpdatedAuditDetails(recordToArchive, userDetails.username, userDetails.msOid, new Date().toISOString(), StatusCode.ARCHIVED);
-
+    newRecord.techRecord_reasonForCreation = 'Update VRM - Cherished Transfer';
```
Comment (shivangidas) [inline src/processors/processVrmRequest.ts:16]: should be `updatedNewRecord`

File: src/processors/processVrmRequest.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const processPatchVrmRequest = (currentRecord: TechRecordGet, userDetails
...
     return [updatedRecordToArchive, newRecord];
```
Comment (shivangidas) [inline src/processors/processVrmRequest.ts:17]:
```suggestion
    return [updatedRecordToArchive, updatedNewRecord];
```

File: src/processors/processVrmRequest.ts (hunk 1 above)
Comment (shivangidas) [reply src/processors/processVrmRequest.ts:16]: That part is fine, there's a `updatedNewRecord` with the audit details and new vrm, that's the one you would want to use. (There's `updatedRecordToArchive` for the archive one)

File: src/processors/processVrmRequest.ts (hunk 2 above)
Comment (owen-corrigan-bjss) [reply src/processors/processVrmRequest.ts:17]: Sorry I completely misread that. I'll sort it now

**Review by andrewf-bjss — APPROVED** [review]

**Pull request merged**